assets/js/src/config.js

```diff
@@ -1,11 +1,7 @@
 /* global wcgaiData */
 /* eslint-disable camelcase */
-export const {
-	config,
-	events,
-	cart,
-	products,
-	product,
-	added_to_cart: addedToCart,
-	order,
-} = wcgaiData;
+export const eventData = function() {
+	return wcgaiData;
```

Comment:
What's the point fo having this as a function? instead of returning it directly?